CHANGELOG.md

```diff
@@ -8,11 +8,16 @@ and this project adheres to
 
 ## [Unreleased]
 
+---
+
+## [0.16.5] - 2024-07-10
+
 ### Security
 
 - Switched to using XChaCha20Poly1305 for the redis token cache encryption.
 
-This addresses a few medium severity security issues with the tokens
+This addresses a few medium severity security issues with the tokens. Note that
```

Comment:
Doesn't this imply a breaking change...?

Reply:
I wouldn't say so. You can't use previous versions of the library alongside the new one, but you can upgrade without any changes on your end

Reply:
I see, although I agree with Simone's comment below